critiquebrainz/frontend/templates/review/revisions.html

```diff
@@ -32,7 +32,7 @@
         <tbody id="results">{% include 'review/revision_results.html' %}</tbody>
       </table>
       {% if count > limit %}
-        <button id="more-button" type="button" class="btn btn-primary" onclick="load_more();" >{{ _('Load older revisions') }}</button>
+        <button id="more-button" type="button" class="btn btn-primary" onclick="load_more()">{{ _('Load older revisions') }}</button>
```

Comment:
Why did you add `onclick` back instead of using `click()` method on `$("#more-button")`?